### ISSUE TITLE
Transparent plots make time slider $time=0 in parallel.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1914]
Fady Najjar reported a problem where the time slider in saved images was always showing "Time=0". I did some investigation and was able to determine that when VisIt is run in parallel, transparent plots cause the time value used by the time slider to always be zero. When there are no transparent plots, the time slider shows the right values.

Here is a script that reproduces the problem:

# visit -cli -nowin -np 2 -l srun -s reproduce_timeslider.py
OpenDatabase("/usr/gapps/visit/data/multi_curv2d.silo")
AddPlot("Pseudocolor", "d", 1, 1)
AddOperator("Elevate", 1)
PseudocolorAtts = PseudocolorAttributes()
PseudocolorAtts.colorTableName = "hot_desaturated"
SetPlotOptions(PseudocolorAtts)
DrawPlots()

timeslider = CreateAnnotationObject("TimeSlider")

# Save a good image
SaveWindow()

# Save a bad image (transparency in parallel to blame)
PseudocolorAtts.opacity = 0.411765
SetPlotOptions(PseudocolorAtts)
DrawPlots()
SaveWindow()

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 11/13/2012 06:47 pm
Original update: 12/10/2012 05:23 pm
Ticket number: 1239
